app/components/TablePaginationControls.tsx

```diff
@@ -6,28 +6,39 @@ import ButtonToolbar from "react-bootstrap/ButtonToolbar";
 import AlignedLabel from "./AlignedLabel";
 import { QueryConfig } from "@/utils/queryBuilder";
 import "react-datepicker/dist/react-datepicker.css";
-import { FaCircleArrowRight, FaCircleArrowLeft } from "react-icons/fa6";
+import { FaAngleLeft, FaAngleRight } from "react-icons/fa6";
+import { HasuraAggregateData } from "@/types/graphql";
 
 interface PaginationControlProps {
   queryConfig: QueryConfig;
   setQueryConfig: Dispatch<SetStateAction<QueryConfig>>;
   recordCount: number;
   isLoading: boolean;
+  aggregateData?: HasuraAggregateData;
 }
 
 /**
  * UI component that controls pagination by setting the
  * QueryConfig offset
  */
 export default function TablePaginationControls({
+  aggregateData,
   queryConfig,
   setQueryConfig,
   recordCount,
   isLoading,
 }: PaginationControlProps) {
   const currentPageNum = queryConfig.offset / queryConfig.limit + 1;
+  const totalRecords = aggregateData?.aggregate?.count || 0;
+
   return (
     <ButtonToolbar>
```

Comment:
To your questions about visual style, I'm a fan of it just the way you have it.

The interactive bits are blue (the most classic link-color) and they are using a single character to represent the action, in a light stroke. It's so direct, it's so correct. 

 As many labels, borders, colors, decorations, nicknacks, and general clutter that can be left behind -- the better.